server/src/FaceRest.cc

```diff
@@ -280,8 +280,10 @@ FaceRest::~FaceRest()
 
 	MLPL_INFO("FaceRest: stop process: started.\n");
 	if (m_impl->soupServer) {
+#ifndef SOUP_VERSION_2_48
```

Comment:
それ自体がSince 2.42だから、ですかねぇ

https://developer.gnome.org/libsoup/stable/libsoup-2.4-Version-Information.html#SOUP-CHECK-VERSION:CAPS

HatoholではSOUP_VERSION_2_32というのも使ってるし。